src/node.cc

```diff
@@ -120,6 +120,29 @@ typedef int mode_t;
 extern char **environ;
 #endif
 
+/* TODO: remove this if the Interceptors are supported */
```

Comment:
What about using `TODO(V8JERRY)`? In this case all the Node.js modifications can be found easily.